ci-operator/step-registry/ipi/conf/telemetry/ipi-conf-telemetry-commands.sh

```diff
@@ -4,6 +4,18 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
+if test -z "${TELEMETRY_ENABLED}"
+then
+	if [[ "${JOB_NAME}" =~ (^|[^[:digit:]]+)4.11([^[:digit:]]+|$) ]]
```

Comment:
Ok having this off on 4.12 minors and multi-upgrades too?

Reply:
All relevent versions of the monitoring stack understand:

```yaml
telemeterClient:
  enabled: false
```

The issue is whether the origin suite understands that means Telemetry has been disabled.  And 4.11->4.12 update jobs are controlled by a 4.12 origin suite, so they're fine with Telemetry disabled.  The current `JOB_NAME` matching will disable it for those jobs by default, because they mention `4.11`, but :shrug:, we don't _need_ to check Telemetry in those runs, and if we decide we do, we can go for a fancier regexp or explicitly set `TELEMETRY_ENABLED: "true"` for those jobs.